### PR TITLE
Added Gradle and Compile category

### PR DESCRIPTION
**Pull request by Aracem**
Added Gradle and Compile category. Added SpeedUp debug compile time

@FineCinnamon/owners please +1 to accept

### CODE REVIEW

**Comment by lgvalle** [comment]
:+1:

**Comment by pedrovgs** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -25,6 +25,11 @@ _If you collaborate in this repo, please always sort the gist in *Alphabetical O
 [Material Colors] (https://gist.github.com/daniellevass/b0b8cfa773488e138037)
 * Material colors. Colors everywhere
 
+## Gradle & Compile
+
+[Speddup debug compile time] (https://gist.github.com/Aracem/a2bcdb841d9d2df07c23)
```
Comment (xrigau) [inline README.md:30]: do you mean `Speed up`? :smile_cat:

**Comment by Aracem** [comment]
One question, does github has something similar to Bitbucket to prompt someone to "accept" de pull-request?

File: README.md (hunk 1 above)
Comment (pedrovgs) [inline README.md:30]: hahaha nice catch!!!

Comment (Aracem) [inline README.md:30]: hahaha i don't know write :dancer: please fix in the next commit :)

**Comment by xrigau** [comment]
I don't think so - I think you can assign a PR to someone but it doesn't actually prompt him to accept/merge/review it

**Comment by lgvalle** [comment]
Nop, you need to use conventions, as `:+1`

**Pull request merged**